### PR TITLE
Improved the getting started guide

### PR DESCRIPTION
**Pull request by mattpocock**

### CODE REVIEW

File: docs/guides/start.md [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
+## Our first machine
+
 Suppose we want to model a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) as a state machine. First, install XState using NPM or Yarn:
 
 ```bash
 npm install xstate --save
 ```
 
-Then, in your project, import `createMachine`, which is a factory function that creates a state machine or statechart:
+> If you're using VSCode, you should install our [VSCode Extension](https://marketplace.visualstudio.com/items?itemName=mattpocock.xstate-vscode) to allow you to visualise the machine you're building as you go.
```
Comment (laurakalbag) [inline docs/guides/start.md:11]: visualise -- should be -> visualize

**Review by laurakalbag — CHANGES_REQUESTED** [review]
Great update. Just one British spelling slipped through!

File: docs/guides/start.md (hunk 1 above)
Comment (mattpocock) [reply docs/guides/start.md:11]: Gah

Comment (mattpocock) [reply docs/guides/start.md:11]:
```suggestion
> If you're using VSCode, you should install our [VSCode Extension](https://marketplace.visualstudio.com/items?itemName=mattpocock.xstate-vscode) to allow you to visualize the machine you're building as you go.
```

**Review by laurakalbag — APPROVED** [review]

**Pull request merged**